performant-software/Annotation-Studio
# Modify user filtering by rep group

## What does this PR do?
1. User filter menu should only include members of the Anthology rep_group

## How to test?
1. Go to http://cove-staging.herokuapp.com/documents/new-machine-stops using your admin login
2. In the filter by users dropdown, all users who are tagged with any rep_group that the document has been tagged with should be shown.
3. Then go to http://cove-staging.herokuapp.com/anthologies/literature-201/documents/new-machine-stops
4. In the filter by users dropdown, all the users in http://covestaging.herokuapp.com/anthologies/literature-201/?tab=users should be shown.
